### PR TITLE
WL 21523 - Update text in the "How to Quit The Sandbox" Prompt

### PR DESCRIPTION
**Pull request by cain-kilgore**
## Test Plan
1. Install the PR
2. With the Interface still running, launch the installer again and confirm it shows a message similar to '__Interface cannot be installed while the Interface is running. Please close it in the task bar and click Retry to continue.__`
3. With the Interface still running, launch the uninstaller and confirm it shows a message similar to '__Interface cannot be uninstalled while the Interface is running. Please close it in the task bar and click Retry to continue.__`
4. With the Sandbox still running, launch the Installer again and confirm that it shows a message similar to '__Sandbox cannot be installed while Sandbox is running. Please close it in the system tray and click Retry to continue.__'.
5. With the Sandbox still running, launch the uninstaller and confirm that it shows a message similar to '__Sandbox cannot be uninstalled while Sandbox is running. Please close it in the system tray and click Retry to continue.__'.

[WL 21523](https://worklist.net/21523)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3048/HighFidelity-Beta-PR11302-99a99ce71b8b5ed049107d6cf5c1115431e12d55.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3048/HighFidelity-Beta-PR11302-99a99ce71b8b5ed049107d6cf5c1115431e12d55.exe

**Comment by Nex-Pro** [comment]
Testing this

**Comment by Nex-Pro** [comment]
## PASS
Tested on Win10 
Results identical to above.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3075/HighFidelity-Beta-PR11302-e9f666d970488c1e755a31751144d88b90940177.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3075/HighFidelity-Beta-PR11302-e9f666d970488c1e755a31751144d88b90940177.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3077/HighFidelity-Beta-PR11302-4d0df705a9a008176c91aecab19271c8141f80c2.dmg

**Comment by cain-kilgore** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3078/HighFidelity-Beta-PR11302-4d0df705a9a008176c91aecab19271c8141f80c2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3079/HighFidelity-Beta-PR11302-14e5cf31f220db9f0e04566f341d04cbd4bef6ac.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3079/HighFidelity-Beta-PR11302-14e5cf31f220db9f0e04566f341d04cbd4bef6ac.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3080/HighFidelity-Beta-PR11302-2a6b5235426289bbd3c84219fc0f79fdf4e8e39e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3080/HighFidelity-Beta-PR11302-2a6b5235426289bbd3c84219fc0f79fdf4e8e39e.exe

**Comment by cain-kilgore** [comment]
@misslivirose: This has been changed in the next commit. Sandbox and Interface will show separate messages now. :)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3081/HighFidelity-Beta-PR11302-d429bd0b01b585b076470e1e090c7e5abb10103a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3081/HighFidelity-Beta-PR11302-d429bd0b01b585b076470e1e090c7e5abb10103a.exe

**Comment by Nex-Pro** [comment]
re-testing

**Comment by Nex-Pro** [comment]
### Pass On windows.

Interface gives messages referring to taskbar, sandbox messages refer to system tray.

**Comment by cain-kilgore** [comment]
Great, thanks @Nex-Pro!

File: cmake/templates/NSIS.template.in [hunk 1]
```diff
@@ -963,9 +963,18 @@ SectionEnd
     ${If} $R0 == 0
 
       ; the process is running, ask the user to close it
+      
+      ${If} "${displayName}" == "Sandbox"
```
Comment (birarda) [inline cmake/templates/NSIS.template.in]: Change "Sandbox" to "@CONSOLE_DISPLAY_NAME"

File: cmake/templates/NSIS.template.in [hunk 2]
```diff
@@ -963,9 +963,18 @@ SectionEnd
...
       MessageBox MB_RETRYCANCEL|MB_ICONEXCLAMATION \
-        "${displayName} cannot be ${action} while ${displayName} is running.$\r$\nPlease close it and click Retry to continue." \
+        "${displayName} cannot be ${action} while ${displayName} is running.$\r$\nPlease close it in the system tray and click Retry to continue." \
         /SD IDCANCEL IDRETRY Prompt_${UniqueID} IDCANCEL 0
+      ${EndIf}
+
+      ${If} "${displayName}" == "Interface"
```
Comment (birarda) [inline cmake/templates/NSIS.template.in]: Change "Interface" to "@INTERFACE_DISPLAY_NAME@"

File: cmake/templates/NSIS.template.in (hunk 1 above)
Comment (cain-kilgore) [reply cmake/templates/NSIS.template.in]: @CONSOLE_DISPLAY_NAME@ or @CONSOLE_DISPLAY_NAME? (Just looking at the other Interface comment and it doesn't match the format)

**Comment by birarda** [comment]
We should re-QA this to ensure the behaviour is the same after my requested changes. Thanks @cain-kilgore!

**Comment by cain-kilgore** [comment]
No worries!

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3120/HighFidelity-Beta-PR11302-84f37b9663041a1e3b717c1c160fa07790ff4b56.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3120/HighFidelity-Beta-PR11302-84f37b9663041a1e3b717c1c160fa07790ff4b56.dmg

**Comment by ShoaibSmith** [comment]
Testing

**Comment by ShoaibSmith** [comment]
Pass

With the Interface still running, launch the installer again and confirm it shows a message similar to 'Interface cannot be installed while the Interface is running. Please close it in the task bar and click Retry to continue.` - **Pass**
With the Interface still running, launch the uninstaller and confirm it shows a message similar to 'Interface cannot be uninstalled while the Interface is running. Please close it in the task bar and click Retry to continue.` - **Pass**
With the Sandbox still running, launch the Installer again and confirm that it shows a message similar to 'Sandbox cannot be installed while Sandbox is running. Please close it in the system tray and click Retry to continue.'. - **Pass**
With the Sandbox still running, launch the uninstaller and confirm that it shows a message similar to 'Sandbox cannot be uninstalled while Sandbox is running. Please close it in the system tray and click Retry to continue.'. - **Pass**

**Comment by cain-kilgore** [comment]
Thanks @ShoaibSmith !

**Pull request merged**